docs/api/version-history.md

```diff
@@ -26,6 +26,10 @@ keywords: "API, Docker, rcli, REST, documentation"
   `BindOptions.ReadOnlyNonRecursive` and `BindOptions.ReadOnlyForceRecursive` to customize the behavior.
 * `POST /containers/create` now accepts a `HealthConfig.StartInterval` to set the
   interval for health checks during the start period.
+* `POST /containers/create` accepts multiple `EndpointSettings` for a long but
+  this was erroring out prior to API v1.44. Starting with this version, the API
+  will now fulfill such requests and effectively connect containers to multiple
+  networks without needing to submit extra `/networks/{id}/connect` requests.
```

Comment:
Looks like a partial sentence here ("for a long").


Perhaps this is not the place to document that it was technically possible already for a long time, and just document "what's new";

```suggestion
* `POST /containers/create` now accepts multiple networks in 
  `NetworkingConfig.EndpointSettings` now accepts. Starting with this version, the API
  ow fulfills such requests and effectively connect containers to multiple
  networks without needing to submit extra `/networks/{id}/connect` requests.
```

Does this change also affect `docker service create`?

Reply:
Oh, actually; it looks like we may need some changes there (code for swarm services); see https://github.com/moby/moby/blob/462d6ef826861fad021fb565c0481fb61d2db6bc/daemon/cluster/executor/container/container.go#L506-L537

Reply:
As said during our last maintainer's meeting, I won't implement that right away. First, I need to submit a PR to include the Priority field to swmarkit's protobuf to make sure containers are connected to their network in the right order. Second, I'd prefer to see whether that works as expected from the CLI and Compose.